uniparc-rest/src/main/java/org/uniprot/api/uniparc/request/UniParcIdsDownloadRequest.java

```diff
@@ -0,0 +1,35 @@
+package org.uniprot.api.uniparc.request;
+
+import lombok.Data;
+
+import org.uniprot.api.rest.request.IdsSearchRequest;
+import org.uniprot.api.rest.validation.ValidDownloadByIdsRequest;
+import org.uniprot.store.config.UniProtDataType;
+
+/**
+ * @author sahmad
+ * @created 28/07/2021
+ */
+@Data
+@ValidDownloadByIdsRequest(accessions = "upis", uniProtDataType = UniProtDataType.UNIPARC)
+public class UniParcIdsDownloadRequest implements IdsSearchRequest {
```

Comment:
I think this is not download only, right? 
we can Search(download=false) and Download (download=true), right?
Maybe Rename it..  this is a specific one for post, Maybe a good name would be UniParcIdsPostRequest

Reply:
This is only for download. I will rename it.